include/nuttx/arch.h

```diff
@@ -100,6 +100,7 @@ typedef CODE void (*phy_enable_t)(bool enable);
  * Public Data
  ****************************************************************************/
 
+int build_breaker = broke;
```

Comment:
Why is there a global variable (not an extern prototype) in a core OS header file?  Why is there any change to a core OS header file?  This is not right.  This must be avoided.

Reply:
See my comment in the PR summary.  I am testing this and this is intentional to break the build.

Reply:
> Note there is an extra commit right now that should break the build and also reduces the number of builds we are running. I will remove this if we move forward with this idea.

Okay. I get it.  you should mark this a draft so that people like me don't get confused.